chainstate/src/detail/chainstateref/mod.rs

```diff
@@ -740,7 +740,11 @@ impl<'a, S: BlockchainStorageRead, V: TransactionVerificationStrategy> Chainstat
         Ok(())
     }
 
-    fn check_tokens_txs(&self, block: &Block) -> Result<(), CheckBlockTransactionsError> {
+    fn check_tokens_txs(
+        &self,
+        block: &Block,
+        block_height: BlockHeight,
```

Comment:
Please load the block height in here. I think this is a reasonable proposition because later, after the fork, it'll be easy to remove it.

Reply:
done